validate.py

```diff
@@ -213,6 +213,19 @@ def determine_if_mergeable():
   for user, state in sorted(reviews.items()):
     print ('  %s: %s' % (user, state))
 
+  # As the PR gets older, require more and more rejections to "veto" it,
+  # overriding any non-unanimous majority approvals.
+  # Thus, a newly opened PR can be blocked with a single veto (0 days + 1)
+  # A PR that has been open for 3 days requires four rejections to be vetoed.
+  # (Note that this is the number of days since the PR was opened, not since its last change.)
+  pr_creation_timestamp = pr_created_at_ts(pr_json)
```

Comment:
Wait, this should be `pr_last_changed_ts`

Reply:
What's the point of vetoing then?  Why not remove the requirement entirely?

Reply:
Two things:

1. It allows a player to basically demand time to do a more thorough inspection of a PR. If I see something shifty in the morning before I have to run to work, I can reject a PR so I can take a look at it in the evening, even if the rest of the group approves it.

2. It allows a player to basically buy time to argue against a PR; it's sort of analogous to a filibuster. If everyone decides a PR is 👍, but I feel very strongly about not merging it in, I can block it for at least some amount of time while I make my case.

Reply:
In the case of a single rejection (now that #44 is merged) what this PR does is just get rid of rejection blocking.  You already can't merge a PR within a day of its creation without 100% agreement.  So neither of your examples make sense to me.

Instead, I think you should just write comments.  No one wants bad PRs merged, so just write things like "let's take our time reviewing this PR, there's risky stuff in it" or "I think this PR has a subtle flaw around timing, but I'm not going to have time to explain it until tomorrow -- please hold off on approving it for now".